code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -122,6 +122,36 @@
 		changeling.add_antag_datum(new_antag)
 	return TRUE
 
+//////////////////////////////////////////////
+//                                          //
+//           OBSESSED                       //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/obsessed
+	name = "Obsessed"
+	persistent = FALSE
+	antag_flag = ROLE_OBSESSED
+	antag_datum = /datum/antagonist/obsessed
+	minimum_required_age = 0
+	protected_roles = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	restricted_roles = list("Cyborg")
```

Comment:
Why is AI considered qualified for obsessed? Is that intended?

Reply:
Not intended